pkg/domain/entities/images.go

```diff
@@ -269,11 +269,21 @@ type ImageImportReport struct {
 	Id string //nolint
 }
 
+// ImageSaveOptions provide options for saving images.
 type ImageSaveOptions struct {
+	// Compress layers when saving to a directory.
 	Compress bool
-	Format   string
-	Output   string
-	Quiet    bool
+	// Format of saving the image: oci-archive, oci-dir (directory with oci
+	// manifest type), docker-archive, docker-dir (directory with v2s2
+	// manifest type).
+	Format string
+	// MultiImageArchive denotes if the created archive shall include more
+	// than one image.
+	MultiImageArchive bool
```

Comment:
This affects behavior of the “tags” parameter of `ImageEngine.Save`, which is not a part of these options… possible but complicated, starting with “this field is difficult to document”.